src/App.css

```diff
@@ -1,8 +1,8 @@
-.App {
+np.App {
   text-align: center;
 }
 
-.App-logo {
+.App-logo1 {
```

Comment:
This will remain `.App-logo`, since the HTML didn't change in `src/App.js`. You have `<img src={logo} className="App-logo" alt="logo" />`, which is good. You can change this back to `.App-logo`.

Reply:
@knnethdw Thx for these comments. it is because I added a new logo file instead of logo so-called it logo1, it works on my end